README.md

```diff
@@ -2,14 +2,18 @@
 The Grand Opinionated AutoTester (GOAT) automatically applies Seiso's standard testing.
 
 ## Getting Started
-1. Create a dictionary text file in `.github/etc/dictionary.txt` (relative to the root of your git repo).
+1. Create a per-repository dictionary (relative to the root of your git repo).
+```bash
+$ mkdir -p .github/etc/
```

Comment:
7 & 8 break the markdown linter: `Dollar signs used before commands without showing output`